en/00_Getting_Started/00_Server_Requirements.md

```diff
@@ -277,7 +277,7 @@ table may be of use:
 | 5.2 +                    | 8.1 - 8.3   |
 | 5.0 - 5.1                | 8.1 - 8.2   |
```

Comment:
Note I intentionally kept these, since the bit above it says
> if you are looking to upgrade PHP on your Silverstripe CMS site, this table may be of use

That implies the table is meant to include the previous major's PHP versions